packages/fxa-auth-server/scripts/paypal-processor.ts

```diff
@@ -16,6 +18,22 @@ const pckg = require('../package.json');
 const config = require('../config').getProperties();
 
 export async function init() {
+  // Load program options
+  program
+    .version(pckg.version)
+    .option('-g, --grace [days]', 'Grace days to allow. Defaults to 1.', '1')
+    .option(
+      '-r, --retries [times]',
+      'Retry attempts to per day. Defaults to 1.',
+      '1'
+    )
+    .option(
+      '-i, --invoice-age [hours]',
+      'How old the invoice must be to get processed. Defaults to 6.',
```

Comment:
Recommend `How old in hours the invoice must be to get processed. Defaults to 6.`